app/models/thredded/messageboard.rb

```diff
@@ -3,11 +3,13 @@ class Messageboard < ActiveRecord::Base
     FILTERS = %w(markdown bbcode)
 
     extend FriendlyId
-    friendly_id :name, use: :slugged
+    friendly_id :slug_candidates,
+                use:            [:slugged, :reserved],
+                # Avoid route conflicts
+                reserved_words: %w(messageboards private-topics autocomplete-users theme-preview)
```

Comment:
Good point, man. Good point. Forgot that that is in the class scope.

If there's a way to accomplish that where it's *not* obnoxious, then I'd lean towards that but if not - don't worry about it.